BangazonAPI/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,280 @@
+                        if (include == "products")
+                        {
+                            Product products = new Product()
+                            {
+                                Id = reader.GetInt32(reader.GetOrdinal("ProductId")),
+                                ProductTypeId = reader.GetInt32(reader.GetOrdinal("ProductTypeId")),
+                                CustomerId = reader.GetInt32(reader.GetOrdinal("CustomerId")),
+                                Price = reader.GetInt32(reader.GetOrdinal("Price")),
+                                Title = reader.GetString(reader.GetOrdinal("Title")),
+                                Description = reader.GetString(reader.GetOrdinal("Description")),
+                                Quantity = reader.GetInt32(reader.GetOrdinal("Quantity"))
+                            };
+                            order.Products.Add(products);
+                        }
+                        else
+                        { }
+                    }
+                    reader.Close();
+                    return Ok(orders);
+                }
+            }
+        }
+
+        [HttpGet("{id}", Name = "GetOrder")]
+        public async Task<IActionResult> Get([FromRoute] int id)
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"SELECT Id, PaymentTypeId, CustomerId FROM [Order] WHERE Id = @id";
+
+                    cmd.Parameters.Add(new SqlParameter("@id", id));
+                    SqlDataReader reader = cmd.ExecuteReader();
+
+                    Order order = null;
+
+                    if (reader.Read())
+                    {
+                        order = new Order
+                        {
+                            Id = reader.GetInt32(reader.GetOrdinal("Id")),
+                            CustomerId = reader.GetInt32(reader.GetOrdinal("CustomerId"))
+                        };
+                        if (!reader.IsDBNull(reader.GetOrdinal("PaymentTypeId")))
+                        {
+                            order.PaymentTypeId = reader.GetInt32(reader.GetOrdinal("PaymentTypeId"));
+                        }
+                    }
+                    reader.Close();
+                    return Ok(order);
+                }
+            }
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Post([FromBody] Order order)
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = "INSERT INTO [Order] (PaymentTypeId, CustomerId) OUTPUT INSERTED.Id VALUES (@PaymentTypeId, @CustomerId)";
+                    cmd.Parameters.Add(new SqlParameter("@PaymentTypeId", order.PaymentTypeId));
```

Comment:
When you POST, `PaymentTypeId` should always be `null` to start out with. (Every order is unpaid when we create it.) You don't need to use SQL parameters-- the body of the POST shouldn't include `PaymentTypeId` at all, so you can just hard code it into the SQL command